test/NVQPP/cudaq_observe.cpp

```diff
@@ -7,6 +7,8 @@
  ******************************************************************************/
 
 // RUN: nvq++ %s -o %basename_t.x --target quantinuum --emulate && ./%basename_t.x | FileCheck %s
+// RUN: nvq++ %s -o %basename_t.x --target iqm --iqm-machine Adonis --emulate && ./%basename_t.x | FileCheck %s
+// RUN: nvq++ %s -o %basename_t.x --target iqm --iqm-machine Apollo --emulate && ./%basename_t.x | FileCheck %s
```

Comment:
Subtle. Maybe a comment that the two lines test different machines (Adonis and Apollo) for the casual skimmers among us? :)

Reply:
Comment added in ec9ad891